Source/FakeItEasy.Tests/Core/ArgumentValueFormatterTests.cs

```diff
@@ -139,7 +141,25 @@ public void Should_prefer_exact_type_formatter_to_interface_formatter()
             var result = this.formatter.GetArgumentValueAsString("string value");
 
             // Assert
-            Assert.That(result, Is.EqualTo("a string"));
+            result.Should().Be("a string");
+        }
+
+        [Test]
+        public void Provided_formatters_should_have_negative_priority()

```

Comment:
Hmm. I was off reversing the condition, to force a failure, and got

> Expected collection to be empty because no formatters should have non-negative priority, but found 2.

Did you run your example in Linqpad, using future FluentAssertions?